papers/H0_I/Analysis/CRACO/build_craco_H0_Emax_cube.src

```diff
@@ -0,0 +1,10 @@
+python py/craco_H0_Emax_cube.py -n 1 -m 250 -o Cubes/craco_H0_Emax_cube1.out --clobber &
```

Comment:
It would be good in future to get a script that looks at the cube files to determine how many jobs are needed and does the spamming automatically. That's a future to-do however.

Reply:
I think this is actually done in the Analysis/build_build.py script!